di_website/templates/404.html

```diff
@@ -1,9 +1,39 @@
 {% extends 'layouts/default.html' %}
 
-{% block body_class %}template-404{% endblock %}
 
 {% block body %}
-    <h1>Page not found</h1>
 
-    <h2>Sorry, this page could not be found.</h2>
+{% load contactus %}
+
+<section class="hero hero--minor">
+    <span class="hero__flourish"></span>
+    <span class="hero__flourish"></span>
+    <span class="hero__flourish"></span>
+    <span class="hero__flourish"></span>
+    <span class="hero__flourish"></span>
+    <span class="hero__flourish"></span>
+    <span class="hero__flourish"></span>
+    <span class="hero__flourish"></span>
+    <div class="row row--narrow">
+        <div class="hero__content">
+            <h1 class="hero__title"><span>There's been an error</span></h1>
+            <p class="hero__excerpt"><span>What you were looking for cannot be found</span></p>
+        </div>
+    </div>
+</section>
+
+<div class="section">
+    <div class="row row--narrow">
+        <div class="is-typeset as-wysiwyg max-meter">
+            <p>Sorry about that. Please use the navigation above to explore the website or contact us if you still can’t find what you’re looking for.</p>
+            <h4>Some things that might be helpful</h4>
+            <ul>
+                <li>Report this link as broken by emailing <a href="mailto:info@devinit.org">info@devinit.org</a></li>
+                <li><a href="/" rel="home">Our homepage</a></li>
+                <li>{% load_contactus %}</li>
```

Comment:
Could you move the `<li></li>` into the template tag inside of the `if` check? Otherwise if there is no contactus_page, we get a blank bullet point.

Reply:
Just moved the `<li>` and pushed